rider/src/main/kotlin/com/jetbrains/rider/plugins/unity/actions/UnityActions.kt

```diff
@@ -12,7 +12,8 @@ class PlayInUnityAction() : ToggleAction("Play/Edit", "Change Play/Edit mode in
 
     override fun isSelected(e: AnActionEvent):Boolean {
```

Comment:
I'm fine with changes in the following actions, but please merge with the latest master, there were some changes made by me.